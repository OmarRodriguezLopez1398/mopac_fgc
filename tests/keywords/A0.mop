A0
comment1
comment2
C  0.00 0.0 0.0
O  2.19 0.0 0.0
O -2.19 0.0 0.0
