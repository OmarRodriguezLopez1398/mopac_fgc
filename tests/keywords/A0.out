 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.298M, James J. P. Stewart,   **
 **                           web-site: HTTP://OpenMOPAC.net.                 **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Fri Oct 29 13:45:08 2021  *
 *  A0         - INPUT GEOMETRY IS IN ATOMIC UNITS (A0)
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *******************************************************************************
A0
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          0.00000000  *   0.00000000  *   0.00000000  *
     2       O          1.15889809  *   0.00000000  *   0.00000000  *
     3       O         -1.15889809  *   0.00000000  *   0.00000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         C          0.0000      0.0000      0.0000
     2         O          1.1589      0.0000      0.0000
     3         O         -1.1589      0.0000      0.0000

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: C O2  =     3 atoms



      MOLECULAR POINT GROUP   :   D*h 


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 8


          Geometry optimization using EF

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    34.076 HEAT: -84.29628
 CYCLE:     2 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    10.424 HEAT: -84.49421
 CYCLE:     3 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.226 HEAT: -84.51541

     GRADIENT =  0.22636 IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
 A0
 comment1
 comment2


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.298M)
                                                       Fri Oct 29 13:45:08 2021




          FINAL HEAT OF FORMATION =        -84.51541 KCAL/MOL =    -353.61246 KJ/MOL


          COSMO AREA              =         65.56 SQUARE ANGSTROMS
          COSMO VOLUME            =         47.24 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.22636          =       0.13069 PER ATOM
          IONIZATION POTENTIAL    =         12.759299 EV
          HOMO LUMO ENERGIES (EV) =        -12.759  0.547
          NO. OF FILLED LEVELS    =          8
          MOLECULAR WEIGHT        =         44.0098         POINT GROUP:  D*h 

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            O     3    O     2     2.33625
            C     1    O     2     0.00000
            O     2    C     1     0.00000


          SCF CALCULATIONS        =          5
 
          WALL-CLOCK TIME         =          0.012 SECONDS
          COMPUTATION TIME        =          0.012 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          0.00000000  *  -0.00000000  *   0.00000000  *
     2       O          1.16812389  *   0.00000000  *  -0.00000000  *
     3       O         -1.16812389  *   0.00000000  *  -0.00000000  *

                             CARTESIAN COORDINATES

   1    C        0.000000000    -0.000000000     0.000000000
   2    O        1.168123885     0.000000000    -0.000000000
   3    O       -1.168123885     0.000000000    -0.000000000


           Empirical Formula: C O2  =     3 atoms



      MOLECULAR POINT GROUP   :   D*h 


                  EIGENVALUES  
 -34.16417 -32.69277 -21.03615 -17.36608 -17.36608 -16.20331 -12.75930 -12.75930
   0.54667   0.54667   1.33449   4.68337


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          C           0.646608        3.3534     1.10761     2.24578
    2          O          -0.323304        6.3233     1.87650     4.44680
    3          O          -0.323304        6.3233     1.87650     4.44680
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000    -0.000     0.000     0.000
 HYBRID         0.000     0.000    -0.000     0.000
 SUM            0.000    -0.000     0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  C   1.10761   0.80552   0.72013   0.72013
    2  O   1.87650   1.16693   1.63994   1.63994
    3  O   1.87650   1.16693   1.63994   1.63994

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.02 SECONDS

 == MOPAC DONE ==
